web/src/SubscriptionEditPage.js

```diff
@@ -46,18 +46,23 @@ class SubscriptionEditPage extends React.Component {
 
   getSubscription() {
     SubscriptionBackend.getSubscription(this.state.organizationName, this.state.subscriptionName)
-      .then((subscription) => {
-        if (subscription === null) {
+      .then((res) => {
+        if (res === null) {
           this.props.history.push("/404");
           return;
         }
 
+        if (res.status === "error") {
+          Setting.showMessage("error", res.msg);
+          return;
+        }
+
         this.setState({
-          subscription: subscription,
+          subscription: res,
```

Comment:
shouldn't it be `res.data`? Did you test all pages to make sure they still work?

Reply:
Yes, test all pages. Backend uses `res.data` whenever using  `c.ResponseOk`. But in these cases, backend uses `c.Data["json"] = subscription` hence `res` also `entity` as well.

Reply:
@thanhtinhpas1 need to wrap all raw data with ResponseOk or ResponseError

Reply:
The impact scope right now will be bigger, is it ok for you?

Reply:
OK